### PR TITLE
feat(platform): Navigation buttons

### PR DESCRIPTION
**Pull request by chargome**
closes https://github.com/getsentry/sentry-docs/issues/11572

### CODE REVIEW

File: src/components/docPage/index.tsx [hunk 1]
```diff
@@ -78,6 +84,16 @@ export function DocPage({
               <div id="main">
                 <CodeContextProvider>{children}</CodeContextProvider>
               </div>
+
+              <div className="grid grid-cols-2 gap-4 not-prose mt-8">
```
Comment (stephanie-anderson) [inline src/components/docPage/index.tsx:88]: just a small nit - would use `mt-16` here to allow for more space between content and navigation

**Comment by stephanie-anderson** [comment]
Haven't had the chance to check - I think this is not up to date with master. Have we checked if hidden pages are properly excluded from that navigation? I can think of the metrics page in develop docs as a good example: 

Screenshot 2024-10-23 at 17 53 19

**Comment by chargome** [comment]
@stephanie-anderson this is up to date with master, metrics is not excluded – or maybe I don't understand the question 🤔

**Comment by stephanie-anderson** [comment]
Oh, this page was never hidden in the first place 🙈 I was referring to whether we checked if the frontmatter config `sidebar_hidden` would be properly respected (as in, it would never be a previous or next page suggestion)

File: src/docTree.ts [hunk 2]
```diff
@@ -146,6 +146,108 @@ export function nodeForPath(node: DocNode, path: string | string[]): DocNode | u
...
+
+  const index = siblings.indexOf(node);
+  if (index > 0) {
+    return siblings[index - 1];
+  }
+
+  return undefined;
+};
+
+const sortSiblingsByOrder = (a: DocNode, b: DocNode) =>
```
Comment (s1gr1d) [inline src/docTree.ts:242]:
```suggestion
const sortBySidebarOrder = (a: DocNode, b: DocNode) =>
```

**Review by s1gr1d — COMMENTED** [review]
Super cool addition to the docs!

**Review by a-hariti — APPROVED** [review]

**Review by stephanie-anderson — APPROVED** [review]

**Pull request merged**